scripts/lintChanged.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Checks for changes between the common ancestor of origin/main and the
+# current HEAD. 
+
+source scripts/shellUtils.sh
+
+MERGE_BASE_SHA_HASH=$(git merge-base origin/main HEAD)
+# Check if output is empty.
+if [ -z "${MERGE_BASE_SHA_HASH}" ]; then
```

Comment:
Prefer double brackets in bash. Also NAB you don't need to put the curly braces around the variable unless they are necessary or you find it improves readability.

```suggestion
if [[ -z "${MERGE_BASE_SHA_HASH}" ]]; then
```